code/controllers/subsystem/traumas.dm

```diff
@@ -46,7 +46,7 @@ SUBSYSTEM_DEF(traumas)
 					   /mob/living/simple_animal/drone, /mob/living/simple_animal/bot, /mob/living/simple_animal/hostile/swarmer)),
 					   "doctors"   = typecacheof(list(/mob/living/simple_animal/bot/medbot)),
 					   "the supernatural"   = typecacheof(list(/mob/living/simple_animal/hostile/construct,
-					   /mob/living/simple_animal/revenant, /mob/living/simple_animal/shade)),
+					   /mob/living/simple_animal/revenant, /mob/living/simple_animal/hostile/construct/shade)), //SKYRAT EDIT, makes shade a construct
```

Comment:
Have original... This still doesn't conform to regulations. Please look at any other modularisation compliant PR and take from that.

Reply:
The only PR that appears to mess around with lists is https://github.com/Skyrat-SS13/Skyrat-tg/pull/1383/, and it just adds in the things without any comments or such.
Really unsure what you want me to do, simply telling me to look at open PR's isnt really helpfull.
Do I need to perhaps comment out this list and make a whole new one with the additions, or += the new thing in a modular spot?